src/theory/arith/simplex.cpp

```diff
@@ -70,7 +70,6 @@ bool SimplexDecisionProcedure::standardProcessSignals(TimerStat &timer, IntStat&
 
         Debug("recentlyViolated")
           << "It worked? "
-          << conflicts.get()
```

Comment:
Was this deletion intended?

Reply:
At some point, the proxy objects did not have getters. This has changed since, I'm undoing this deletion.